### ISSUE TITLE
bug? GET with simple join, aliasing embedded columns

### ISSUE THREAD

**Issue by rkoberg** [begriffs/postgrest#669]
This URL works as expected:
```
/formats?id=eq.1&select=*,size{*}
```
That is, all columns in the record referenced by size are included

Using this URL also works:
```
/formats?id=eq.1&select=*,size{id,name}
```
That is, the size id and name column result is included.

However, using this URL results in an error:
```
/formats?id=eq.1&select=*,size{value:id,label:name}
```
```
{"hint":null,"details":null,"code":"42703","message":"column sizes_size.id does not exist"}
```
The formats table has a column named ```size_id``` referenceing the ```sizes``` table.

**Comment by ruslantalpa** [comment]
does this work?
```
/formats?id=eq.1&select=*,size{id,label:name}
```

**Comment by rkoberg** [comment]
Yes, that works (aliasing name with label)

**Comment by ruslantalpa** [comment]
short story is that when embedding a parent, you always have to include it's primary key and as it looks, you can't `rename` it.
It's kind of a limitation/bug

**Comment by rkoberg** [comment]
OK, "Fact of Life" issue. Closing...

**Comment by steve-chavez** [comment]
This was fixed in https://github.com/begriffs/postgrest/pull/978 and released in [v0.4.4.0](https://github.com/begriffs/postgrest/releases/tag/v0.4.4.0).
